### PR TITLE
CrazyGames documentation improvement

### PR DESCRIPTION
**Pull request by Bouh**
- Add a link to the project example "CrazyGames SDK demo".
- Add new details and tips about the conditions and actions from the CrazyGames SDK.

### CODE REVIEW

File: docs/gdevelop5/extensions/crazy-games-ad-api/details.md [hunk 1]
```diff
@@ -1,24 +1,84 @@
...
+
+Some actions or events triggered automatically can be used to track when and how users are playing your games and get the stats directly on CrazyGames game dashboard.
+
+- The **Gameplay Start** action has to be called whenever the player starts playing or resumes playing after a break (menu/loading/achievement screen, game paused, etc.).
+
+- The **Gameplay Stop** action has to be called on every game break (entering a menu, switching levels, pausing the game, etc.). Don't forget to call the Gameplay Start when the gameplay resumes.
+
+### Invite Button
+
+DIsplay on CrazyGames website a popup containing the invite link with a copy button, very useful for multiplayer games to give a friend the possibility to join a session quickly.
```
Comment (D8H) [inline docs/gdevelop5/extensions/crazy-games-ad-api/details.md:71]: The sentence is too long.
```suggestion
Display, on CrazyGames website, a popup containing the invite link with a copy button, very useful for multiplayer games to give a friend the possibility to join a session quickly. 
```

**Comment by D8H** [comment]
@Bouh Can you give a link to the page? It's useful for reviewers.

**Comment by Bouh** [comment]
> @Bouh Can you give a link to the pages? It's useful for reviewers.

https://gdevelop-wiki-git-fork-bouh-crazygames-documentation-gdevelop.vercel.app/gdevelop5/publishing/crazy-games/

Then

https://gdevelop-wiki-git-fork-bouh-crazygames-documentation-gdevelop.vercel.app/gdevelop5/extensions/crazy-games-ad-api/details/

File: docs/gdevelop5/extensions/crazy-games-ad-api/details.md [hunk 2]
```diff
@@ -1,24 +1,84 @@
 ---
 title: Details
 ---
-# CrazyGames SDK extension
+# CrazyGames SDK Extension
 
-This extension allows games to be hosted on CrazyGames website and display ads.
+This extension allows games to be hosted on the CrazyGames website and display modules in games such as ads, banners, and various functionality related to the game.
```
Comment (4ian) [inline docs/gdevelop5/extensions/crazy-games-ad-api/details.md:6]:
```suggestion
This extension allows games to be hosted on the CrazyGames website and display modules in games such as ads, banners, and various functionalities offered by CrazyGames.
```

File: docs/gdevelop5/extensions/crazy-games-ad-api/details.md [hunk 3]
```diff
@@ -1,24 +1,84 @@
...
 
-!!! tip
+## Advanced Configurations
+
+Please refer to the [CrazyGames SDK documentation](https://docs.crazygames.com/sdk/html5/intro/) to learn how to set up compatible environments for games and to get tips about unwanted scrolling on the game page.
+
+## Demo
```
Comment (4ian) [inline docs/gdevelop5/extensions/crazy-games-ad-api/details.md:12]:
```suggestion
## Try the CrazyGames SDK demo
```

File: docs/gdevelop5/extensions/crazy-games-ad-api/details.md [hunk 4]
```diff
@@ -1,24 +1,84 @@
...
+
+A project demo showcase the features from the extension in GDevelop.
```
Comment (4ian) [inline docs/gdevelop5/extensions/crazy-games-ad-api/details.md:14]:
```suggestion
A demo project showcases the features from the CrazyGames SDK in GDevelop.
```

File: docs/gdevelop5/extensions/crazy-games-ad-api/details.md [hunk 5]
```diff
@@ -1,24 +1,84 @@
...
+The project can be exported as HTML5 files, then imported in the QA Tool on CrazyGames website when submitting a new game on their platform.
```
Comment (4ian) [inline docs/gdevelop5/extensions/crazy-games-ad-api/details.md:15]:
```suggestion
The project can be exported as an HTML5 game, then imported in the QA Tool on CrazyGames website when submitting a new game on their platform.
```

File: docs/gdevelop5/extensions/crazy-games-ad-api/details.md [hunk 6]
```diff
@@ -1,24 +1,84 @@
...
 
-## Display a video ad
+[Open the demo](https://gdevelop.io/fr-fr/game-example/free/crazygames-sdk-demo){ .md-button .md-button--primary }
 
-Ads can be displayed using the **Display video ad** action. An ad can either be `"midgame"` or `"rewarded"`. Please refer to [CrazyGames' ads guidelines](https://docs.crazygames.com/general/ads-guidelines/) to understand when to use one or the other.
+## Display Video Ads
+
+A video ad filling the entire game screen can be displayed using the **Display video ad** action. An ad can either be `"midgame"` or `"rewarded"`.
+
+- **Midgame**: Advertisements can happen when a user dies, a level has been completed, etc.
```
Comment (4ian) [inline docs/gdevelop5/extensions/crazy-games-ad-api/details.md:25]:
```suggestion
- **Midgame**: Show an advertisement when a user dies, a level has been completed, etc.
```

File: docs/gdevelop5/extensions/crazy-games-ad-api/details.md [hunk 7]
```diff
@@ -1,24 +1,84 @@
...
+
+- **Rewarded**: Advertisements can be requested by the user in exchange for a reward (an additional life, a retry when the user dies, a bonus starting item, extra starting health, etc.). Rewarded ads should be shown when users explicitly consent to watch an advertisement.
```
Comment (4ian) [inline docs/gdevelop5/extensions/crazy-games-ad-api/details.md:27]:
```suggestion
- **Rewarded**: An advertisements that can be requested by the user in exchange for a reward (an additional life, a retry when the user dies, a bonus starting item, extra starting health, etc.). Rewarded ads should be shown when users explicitly consent to watch an advertisement.
```

File: docs/gdevelop5/extensions/crazy-games-ad-api/details.md [hunk 8]
```diff
@@ -1,24 +1,84 @@
...
 
 ![](crazy-games-video-ad-action.png)
 
-You can use the **Video ad just finished playing** condition to resume the game after the end of the ad. 
+You can use the **Video ad just finished playing** condition to resume the game after the end of the ad.
 
 ![](crazy-games-video-ad-events.png)
 
+## Banners
+### Request Banner
```
Comment (4ian) [inline docs/gdevelop5/extensions/crazy-games-ad-api/details.md:36]:
```suggestion
### Request a Banner
```

File: docs/gdevelop5/extensions/crazy-games-ad-api/details.md [hunk 9]
```diff
@@ -1,24 +1,84 @@
...
+
+![](ad_example.png)
+
+!!! warning
+
+    A banner can be called only once per 60 seconds. If a banner is called before the delay, an error is returned in the debugger.
+
+### Clearing the Banners
+
+Two possible actions for clearing banners:
```
Comment (4ian) [inline docs/gdevelop5/extensions/crazy-games-ad-api/details.md:48]:
```suggestion
There are two possible actions for clearing banners:
```

File: docs/gdevelop5/extensions/crazy-games-ad-api/details.md [hunk 10]
```diff
@@ -1,24 +1,84 @@
...
+
+### Happy Time
+
+Celebrate a special moment in the game with some sparkling confetti. Celebrate the defeat of a boss, reaching a new high score, or something big.
+
+![](HappyTime.png)
+
+### Tracking Users
+
+Some actions or events triggered automatically can be used to track when and how users are playing your games and get the stats directly on CrazyGames game dashboard.
```
Comment (4ian) [inline docs/gdevelop5/extensions/crazy-games-ad-api/details.md:63]:
```suggestion
Some actions or events are triggered automatically and can be used to track when and how users are playing your games. You can get the stats directly on the CrazyGames game dashboard.
```

File: docs/gdevelop5/extensions/crazy-games-ad-api/details.md [hunk 11]
```diff
@@ -1,24 +1,84 @@
...
+
+- The **Gameplay Start** action has to be called whenever the player starts playing or resumes playing after a break (menu/loading/achievement screen, game paused, etc.).
+
+- The **Gameplay Stop** action has to be called on every game break (entering a menu, switching levels, pausing the game, etc.). Don't forget to call the Gameplay Start when the gameplay resumes.
+
+### Invite Button
+
+Display on CrazyGames website a popup containing the invite link with a copy button, very useful for multiplayer games to give a friend the possibility to join a session quickly.
```
Comment (4ian) [inline docs/gdevelop5/extensions/crazy-games-ad-api/details.md:71]:
```suggestion
Display on CrazyGames website a popup containing the invite link with a copy button. This is very useful in a multiplayer game to give a friend the possibility to join a session quickly. 
```

File: docs/gdevelop5/publishing/crazy-games/index.md [hunk 12]
```diff
@@ -11,12 +11,17 @@ Some games made with GDevelop that have been published on CrazyGames:
 
 ![Ball Challenge 2 on CrazyGames](crazy_games_ball_challenge_2.png)
 
+## How to publish on CrazyGames
 
-!!! tip
+Games can be published on CrazyGames through their website, and benefit of a QA Tool to check the SDK implementation that allow to triggers Ads, Banners, celebrate a win, invite button to various functionality related to the game.
```
Comment (4ian) [inline docs/gdevelop5/publishing/crazy-games/index.md:16]:
```suggestion
Games can be published on CrazyGames through their website. CrazyGames offers a QA Tool to check the implementation of the extension: triggers ads, banners, celebrate a win, show the invite button and all the features offered by CrazyGames for a game.
```

File: docs/gdevelop5/publishing/crazy-games/index.md [hunk 13]
```diff
@@ -11,12 +11,17 @@ Some games made with GDevelop that have been published on CrazyGames:
...
 
+## How to publish on CrazyGames
 
-!!! tip
+Games can be published on CrazyGames through their website, and benefit of a QA Tool to check the SDK implementation that allow to triggers Ads, Banners, celebrate a win, invite button to various functionality related to the game.  
 
-    The [CrazyGames SDK extension](/gdevelop5/extensions/crazy-games-ad-api/details) helps handling ads.
+Check the CrazyGames SDK extension to get helps on all CrazyGames' features.
 
+[CrazyGames SDK extension](/gdevelop5/extensions/crazy-games-ad-api/details){ .md-button .md-button--primary }
```
Comment (4ian) [inline docs/gdevelop5/publishing/crazy-games/index.md:20]:
```suggestion
[Read the CrazyGames SDK extension documentation](/gdevelop5/extensions/crazy-games-ad-api/details){ .md-button }
```

**Review by 4ian — APPROVED** [review]

**Pull request merged**